corehq/apps/userreports/admin.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
Did you use a tool to do this, or did you do it manually? If a tool, please share the command as I have a feeling others will find that useful.

Reply:
```python-modernize corehq/apps/userreports/ -w --future-unicode```

I'd recommend adding a ```--no-six``` in the future